src/main.ts

```diff
@@ -63,7 +63,7 @@ async function run(): Promise<void> {
         await context.asyncForEach(inputs.driverOpts, async driverOpt => {
           createArgs.push('--driver-opt', driverOpt);
         });
-        if (inputs.buildkitdFlags) {
+        if (inputs.driver != 'remote' && inputs.buildkitdFlags) {
```

Comment:
We need the same logic below for `config` as well I think - see [driver/remote/factory.go](https://github.com/docker/buildx/blob/1eff9310f8da42250528bdf0aabcb278d082dae5/driver/remote/factory.go#L46-L51).

Reply:
Have not done it for `config` because it doesn't have a default value like `buildkitdFlags` but maybe we should just skip for it too I'm not sure.

Reply:
Ok actually will be confusing if we do for one but not the other. Should be good now.